docs/hooks/ticketref.md

```diff
@@ -0,0 +1,11 @@
+# Description
+
+Prefixes the commit-msg with the ticket reference, this allows for some workflows to be able to correlate a number of commit's against a specific feature/ticket/incident/issue etc once merged.
+
+a branch of `ABC-1234-hello-world` would turn a commit of `git commit -m "Hello World"` into `[ABC-1234] Hello World`
```

Comment:
This has the big drawback of making the first line longer. The first line is supposed to be less than 50 chars. Would it be possible to produce something like this instead:

```
Hello world

Refs ABC-1234
```